servicebox-app/src/main/java/io/halkyon/model/Service.java

```diff
@@ -0,0 +1,43 @@
+package io.halkyon.model;
+
+import io.quarkus.hibernate.orm.panache.PanacheEntityBase;
+import io.quarkus.panache.common.Sort;
+import org.jboss.resteasy.annotations.jaxrs.FormParam;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+import java.util.List;
+
+@Entity
+public class Service extends PanacheEntityBase {
```

Comment:
Remark: That should not be changed part of this PR but if you prefer to use PanacheEntityBase as class that we extend, then we must also review the Claim class.

Reply:
We need to extend PanacheEntityBase instead to PanacheEntity to be able to custom the Id generation strategy. See [this](https://stackoverflow.com/a/61067172). Before this, we were using the same sequence generators for ids in claims and service table.
To be homogeneous and more PROD friendly I will do the same for Claim entity, wdyt?

Reply:
This is also what I was proposing but maybe using a different PR ...

Reply:
I have already done in the last commit